data/result-exporter/swe-cne-exporter/src/main/java/com/powsybl/openrao/data/swecneexporter/SweAdditionalConstraintSeriesCreator.java

```diff
@@ -75,9 +77,30 @@ private AdditionalConstraintSeries generateAdditionalConstraintSeries(AngleCnecC
             additionalConstraintSeries.setMRID(angleCnecCreationContext.getCreatedObjectId());
             additionalConstraintSeries.setBusinessType(ANGLE_CNEC_BUSINESS_TYPE);
             additionalConstraintSeries.setName(angleCnec.getName());
-            additionalConstraintSeries.setQuantityQuantity(BigDecimal.valueOf(raoResult.getAngle(crac.getInstant(InstantKind.CURATIVE), angleCnec, Unit.DEGREE)).setScale(1, RoundingMode.HALF_UP));
+            additionalConstraintSeries.setQuantityQuantity(roundAngleValue(angleCnec, crac, raoResult));
             return additionalConstraintSeries;
         }
         return null;
     }
+
+    /**
+     * Round the angle value of an AngleCNEC with a relevant number decimals.
+     * Generally, only one decimal suffices but in case of very small violations,
+     * the number of decimals must be increased so the violation can be
+     * read directly in the results.
+     */
+    static BigDecimal roundAngleValue(AngleCnec angleCnec, Crac crac, RaoResult raoResult) {
+        double angle = raoResult.getAngle(crac.getInstant(InstantKind.CURATIVE), angleCnec, Unit.DEGREE);
+        double margin = -Double.MAX_VALUE;
+        for (Threshold threshold : angleCnec.getThresholds()) {
+            Optional<Double> maxValue = threshold.max();
```

Comment:
please do the same as private method computeMargin in AngleCnec and use getLowerBound, getUpperBound